services/essentials.yaml

```diff
@@ -104,6 +104,9 @@ spec:
         - groupId: io.jenkins.plugins
           artifactId: artifact-manager-s3
           version: '1.1'
+        - groupId: org.jenkins-ci.plugins
+          artifactId: node-iterator-api
+          version: 1.5.0
```

Comment:
pulled in by `ec2` plugin. We would ideally IMO need some way to tell the system we're only declaring a given plugin to override the version being pulled in transitively. IOW, in that case, this plugin should be uninstalled if it is not pulled in transitively anymore.